web_transport/sdk/impl/web_transport_factory_impl.cc

```diff
@@ -135,15 +136,7 @@ void WebTransportFactoryImpl::Init() {
   base::CommandLine::Init(0, nullptr);
   base::CommandLine* command_line(base::CommandLine::ForCurrentProcess());
   command_line->AppendSwitch("--quic_default_to_bbr");
-  // Logging settings for Chromium.
-#ifdef _DEBUG
-  logging::SetMinLogLevel(logging::LOG_VERBOSE);
-#else
-  logging::SetMinLogLevel(logging::LOG_WARNING);
-#endif
-  logging::LoggingSettings settings;
-  settings.logging_dest = logging::LOG_TO_STDERR;
-  InitLogging(settings);
+  Logging::InitLogging();
```

Comment:
Why you do this in your library implementation? I think you should instead let application do this.

Reply:
We don't ask developer to init logging module in native SDK either, so it just follows the same workflow. We also don't ask developers to init other modules like SSL. Another reason is this class doesn't expose APIs for `LoggingSettings`. It's configured by SDK.